content_script/core/option.js

```diff
@@ -108,7 +108,7 @@ if (!browser && chrome) {
     option.clear = function () {
         cachedOptions = Object.create(null);
         return new Promise(resolve => {
-            browser.storage.local.clear(resolve);
+            browser.storage.sync.clear(resolve);
```

Comment:
@fztfztfztfzt Seems that Edge does not support `sync` till now. Any reson for using `sync`?